packages/near-fast-auth-signer/src/components/SignMultichain/bitcoin/schema.ts

```diff
@@ -3,15 +3,32 @@ import * as yup from 'yup';
 import { BaseSendMultichainMessageSchema } from '../schema';
 
 export const SendBTCMultichainMessageSchema = BaseSendMultichainMessageSchema.shape({
-  chain:   yup.number().required().test('is-btc', 'Invalid BTC chain value', (value) => value === 0),
-  network: yup.string().oneOf(['mainnet', 'testnet']).required(),
-  fee:     yup.number().optional(),
-  utxos:   yup.array().optional(),
-}).test('fee and utxos check', 'If fee is present, utxos must be present and vice versa', (value) => {
-  const { fee, utxos } = value;
-  // Check if both are present or both are undefined
-  if ((fee && utxos) || (!fee && !utxos)) {
-    return true; // Validation succeeds
+  transaction: yup.object().shape({
+    to:      yup.string().required(),
+    value:   yup.string().required(),
+    inputs:  yup.array().optional(),
```

Comment:
Agree with replacing utxos and fees with inputs and outputs. what do you think about slightly more validation for inputs to help devs catch errors early?

```typescript
inputs: yup.array(yup.object({
  txid: yup.string().required(),
  vout: yup.number().required(),
  amount: yup.number().required()
})).optional()
```

Reply:
Good point, I will narrow the validation